src/main/java/net/mrscauthd/beyond_earth/events/Gravity.java

```diff
@@ -15,41 +15,45 @@ public static void Gravity(LivingEntity entity, GravityType type, Level world) {
         double venus = 0.04;
         double orbit = 0.02;
 
+        if (Methods.isWorld(world, Methods.overworld)) {
+            return;
```

Comment:
Avoid checking if the world is in each space world (e.g. moon, orbit, mars, glacio...) if we know the world is overworld.